examples/Serving-An-Implicit-Model-With-Merlin-Systems.ipynb

```diff
@@ -0,0 +1,451 @@
+{
```

Comment:
<p>Not sure, if we can set <code>use_gpu=True</code> for the different unittest environments. Let's clarify </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/systems/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
discussed on slack, implemented the suggested changes, our test now should be compatible with the current state of the CI, but should also execute both on CPU and GPU when triton is available on the machine (which is not true currently for CPU only systems)